integration/apps/rack/script/ci

```diff
@@ -44,7 +44,8 @@ echo ""
 APP_IMAGE=$APP_IMAGE docker-compose $APP_COMPOSE_FILES build
 
 # Run the test suite
-APP_IMAGE=$APP_IMAGE docker-compose $APP_COMPOSE_FILES run integration-tester
+APP_IMAGE=$APP_IMAGE docker-compose $APP_COMPOSE_FILES run integration-tester || \
+  APP_IMAGE=$APP_IMAGE docker-compose $APP_COMPOSE_FILES logs # Print container logs on `run` failure
```

Comment:
I looked for a solution for `docker-compose run` to simply output its logs, but couldn't find one, so I resorted to two separate commands.

I as looking for something similar to the live tail output of `docker-compose up`, but that doesn't seem to exist for `docker-compose run`.